index.bs

```diff
@@ -5714,9 +5839,14 @@ partial dictionary MLOpSupportLimits {
   </tr>
   <tr>
     <td>{{b}}</td>
-    <td>[=/same as=] {{a}}</td>
+    <td>[=/same type as|same as=] {{a}}</td>
     <td>2 or greater</td>
   </tr>
+  <tr>
+    <td>*output*</td>
+    <td>[=/same type as|same as=] {{a}}</td>
+    <td>maximum of {{a}}'s [=MLOperand/rank=] and {{b}}'s [=MLOperand/rank=]</td>
```

Comment:
Aah, I forgot matmul supports bidirectional broadcasting (whereas gemm does not 🙃).